src/environment/story_describer.py

```diff
@@ -0,0 +1,17 @@
+from typing import List, Optional
+
+from rake_nltk import Rake
+
+
+class StoryDescriber:
```

Comment:
StoryExtractor might be a better name 🤔 It's purpose is to extract data from the story, right?

Reply:
I was looking at location's attribute description when figuring out the name. Can defo be extractor.

Again by oral agreement, we agreed to use the same llama artistic description every time the player will come back to some location that has already been explored. This no longer applies and will be changed in my following PR.